src/gamestate.ts

```diff
@@ -115,12 +115,14 @@ export class GameState {
 
 		try {
 			renderer.render_jays();
-		} catch (err) { }
+		} catch (err) {
+			console.error(err);
+		}
```

Comment:
Ahah en fait non, j'ai volontairement caché le warning inévitable dû au loader X)
Si c'est pour l'afficher autant virer le try catch, cette partie n'est pas plus sensible qu'une autre si ce n'est qu'il y a le warning
On a déjà prévu de changer le loader de toutes façons (avec des promises)

Reply:
Je voulais juste faire disparaître l'erreur de variable non utilisée

Qu'est-ce que tu veux dire par "warning inévitable" ?

Reply:
Je t'en avais parlé, on en reparle plus tard